src/backend/marsha/settings.py

```diff
@@ -214,6 +214,31 @@ class Base(Configuration):
     # Cache
     APP_DATA_CACHE_DURATION = values.Value(60)  # 60 secondes
 
+    # This setting determines if migration 0016 should be run
+    # This migration will look for every uploaded timed text track in the
+    # S3 source bucket and then trigger the lambda to escape them
+    # By default this setting is false but overriden to true in production,
+    # staging and preprod
+    SHOULD_RUN_TIMEDTEXT_TRACKS_ESCAPING_MIGRATION = values.BooleanValue(False)
```

Comment:
I would make it True by default and override it to False only in Development and Test.